ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/Committer.scala

```diff
@@ -93,4 +94,27 @@ private[committer] trait Committer[Submission, PartialResult] {
 
 object Committer {
   type StepInfo = String
+
+  def getCurrentConfiguration(
+      defaultConfig: Configuration,
+      inputState: Map[DamlStateKey, Option[DamlStateValue]],
+      logger: Logger): (Option[DamlConfigurationEntry], Configuration) =
+    inputState
+      .getOrElse(
+        Conversions.configurationStateKey,
```

Comment:
`Conversions.configurationStateKey` should probably be moved into this object as a (package private) constant -- it does not seem to be related to the rest of the methods in `Conversions`.

Reply:
It seems as though it is related to other methods in `Conversions` such as `partyStateKey` and `packageStateKey` each of which returns `DamlStateKey` wrapping. This one just happens to be parameterless.
It is also used in many places throughout the codebase. Moving it and making it private would result in code replication.